integration-tests/src/test/java/com/example/IntegrationTest.java

```diff
@@ -142,6 +142,27 @@
     assertThat(target.subtypeCalled).isTrue();
   }
 
+  /**
+   * [@Inject] Constructors are injected first, followed by fields, and then methods. Fields and methods in
```

Comment:
`{@code Inject @Inject}`, although I don't think Javadoc is useful on tests. I'd rather see this as a line comment inside near the assertion.